src/docbkx/chapter21-transform.xml

```diff
@@ -34,7 +34,7 @@
 			perform rather intricate things like conditioning,
 			looping. It can also be
 			used as a
-			useful way to express small business
+			way to express small business
```

Comment:
I'm not saying the change is wrong, but it changes the meaning of the sentence. _a useful way_ adds that it is practical, good, better, etc., while _a way_ is much more neutral.